.github/ISSUE_TEMPLATE/system-team-qg-checklist.md

```diff
@@ -22,159 +22,73 @@ QG5 Approval: yes/no <!-- Note: Please ask for the approval from Release Managem
-  - [ ] TRG 5.02 latest tag is not used in helm chart be default
-
-  ``` markdown
-  charts/ 
-      chartNameA/
-        Chart.yaml
-        ... 
-      chartNameB/
-        Chart.yaml
-        ...
-  AUTHORS.md 
-  DEPENDENCIES.md 
-  LICENCE 
-  README.md 
-  ```
-  </details>
-
 - [ ] [TRG 5.04](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-04) CPU and memory limits and requests are properly set
 - [ ] [TRG 5.06](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-06) application must be configurable through the Helm chart <!-- every startup configuration aspect of your application must be configurable through the Helm chart (ingress class, tls, labels, annotations, database, secrets, persistence, env variables) -->
 - [ ] [TRG 5.07](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-07) dependencies are present in the `Chart.yaml` they are properly configured
 - [ ] [TRG 5.08](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-08) a product has a single deployable helm chart that contains all components <!--(backend, frontend, etc.) -->
-  <details>
-  <summary>Checks within TRG 5.08</summary>
-
-  - [ ] TRG 5.08 name of the Chart should be just the product-name without prefix or suffix
-  - [ ] TRG 5.08 values file should contain all available variables (even from subcharts) with default values and comments about what they do
-  - [ ] TRG 5.08 helm install command should successfully install the chart to any supported Kubernetes version cluster (without overwriting default values)
-  - [ ] TRG 5.08 helm test runs without errors
-  </details>
-  
 - [ ] [TRG 5.09](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-09) Helm Test running properly
-  <details>
-  <summary>Checks within TRG 5.09</summary>
-
-  - [ ] TRG 5.09 A GitHub action exist which builds or uses the helm chart which gets released
-  - [ ] TRG 5.09 The GitHub action can be triggered manually through Github WebUI manually running a workflow
-  - [ ] TRG 5.09 Helm test verifies that the application is up and running
-  </details>
 - [ ] [TRG 5.10](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-10) Products need to support 3 versions at a time
-  <details>
-  <summary>Checks within TRG 5.10</summary>
-
-  - [ ] TRG 5.10 latest (K8s version 1.25)
-  - [ ] TRG 5.10 latest - 1 (K8s version 1.24)
-  - [ ] TRG 5.10 latest - 2  (K8s version 1.23)
-  </details>
-- [ ] [TRG 5.11](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-11) Upgradeability PRERELEASE
-  <details>
-  <summary>Checks within TRG 5.11</summary>
-
-  - [ ] TRG 5.11 Based on the Helm test workflow, you must provide a GitHub action which takes the latest released helm chart, does an installation of it and then execute the upgrade to the current / new version.
-  </details>
-  
-#### TRG 6 Released Helm Chart 
+- [ ] [TRG 5.11](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-11) Upgrade-ability PRERELEASE
```

Comment:
Upgradeability as one word. Is it still 'PRERELEASE'?

Reply:
Hi @SebastianBezold don´t know, do you have more info´s if it is still fit our agenda?

Reply:
[chore: fix typo + reduce x + removed PRERELEASE](https://github.com/eclipse-tractusx/.github/pull/15/commits/5577a78c452beadae1d5b7887dd6fc0931164617)

removed it

Reply:
current TRG tells us "you must provide a GitHub action which takes the latest released helm chartuse the Helm test workflow"